docker/dsl/build_pipeline.groovy

```diff
@@ -4,7 +4,7 @@ pipelineJob('build-pipeline-job') {
   // From the Jenkins UI, you can easily add your credentials for this repo to the pipeline to test.
   // For an extra challenge, try configuring this job to include your credential object
   // NEVER commit secrets (like credentials) to GitHub!!!
-  def repo = '[insert-your-repo-url]'
+  def repo = '/home/CSCC/pparasuram/repos/docker-class/prakash-infrastructure-jenkins-part2'
```

Comment:
This will work locally, but will not work outside of your local environment.  No deduction